<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DropSetProperties</class>
 <widget class="QWidget" name="DropSetProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>190</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="layoutMain">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>358</width>
        <height>188</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="layoutScrollArea">
       <item row="0" column="0">
        <widget class="QLabel" name="lblID">
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="id">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblType">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="type">
         <item>
          <property name="text">
           <string>Normal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Append</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DESTINY</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblMutexID">
         <property name="text">
          <string>Mutex ID:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="mutexID">
         <property name="maximum">
          <number>2147483647</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblGiftBoxID">
         <property name="text">
          <string>Gift Box ID:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="giftBoxID">
         <property name="maximum">
          <number>2147483647</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblDrops">
         <property name="text">
          <string>Drops:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="DynamicList" name="drops" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblDesc">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="desc"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblConditions">
         <property name="text">
          <string>Conditions:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="DynamicList" name="conditions" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DynamicList</class>
   <extends>QWidget</extends>
   <header>DynamicList.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
